tsconfig.json

```diff
@@ -11,7 +11,8 @@
 		"removeComments": false,
 		"sourceMap": true,
 		"strict": true,
-		"target": "es6"
+		"target": "es6",
+		"lib": ["es6", "dom", "es2017"]
```

Comment:
I'll defer this one to @bitpshr.  One thing I'd make sure is that it won't break GABA while used in the extension,